docs/cse/administration/create-custom-threat-intel-source.md

```diff
@@ -39,23 +42,18 @@ Rule authors can also write rules that look for threat intelligence information
 ### Create a threat intelligence source from CSE UI
 
 1. Click the Content menu and select **Threat Intelligence**.
-1. Click **Add Source** on the **Threat Intelligence** page.  
-    ![threat-intel-page2.png](/img/cse/threat-intel-page2.png)
-1. Click **Custom** on the **Add Source** popup.  
-    ![custom-button.png](/img/cse/custom-button.png)
+1. Click **Add Source** on the **Threat Intelligence** page. <br/><img src={useBaseUrl('img/cse/threat-intel-page2.png')} alt="Threat Intelligence page" width="800"/>
+1. Click **Custom** on the **Add Source** popup. <br/><img src={useBaseUrl('img/cse/custom-button.png')} alt="Custom button" width="600"/>
 1. On the **Add New Source** popup, enter a name, and if desired, a
-    description for the source.  
-    ![add-custom-source.png](/img/cse/add-custom-source.png)
+    description for the source. <br/><img src={useBaseUrl('img/cse/add-custom-source.png')} alt="Add new source" width="600"/>
```

Comment:
this needs to be joined with the previous line

Reply:
Fixed.